kubernetes_env/benchmark/benchmark.py

```diff
@@ -223,11 +261,11 @@ def main(args):
                         type=int,
                         default=5,
```

Comment:
Instead of measuring for just 5 seconds with a high load, I would recommend a lower load over 30 seconds. Will likely give better data.